nmostesting/Config.py

```diff
@@ -146,26 +146,26 @@
 # SDP testing is not concerned with support for specific media parameters, but must include them in the file
 SDP_PREFERENCES = {
```

Comment:
I think this is looking really good...

One concern I have with removing the `video_` and `audio_` prefixes here is if there are parameters that apply to more than one format where the test suite user might want different values for different formats. Examples could include `bit_rate` but no-one is using that for audio right now at least, and `TP` which applies to mux (`video/SMPTE2022_6`) as well as ST 2110-20/-22 video, I think? But should we cross that bridge only later if we need to, perhaps by nesting the format-specific params inside a format or media_type key?

Reply:
Yes, I was thinking that - we could add an audio/video layer to the dict, but not currently needed, and may never be needed - so let's refactor when the need arises.